### PR TITLE
Fix README docs

### PR DESCRIPTION
**Pull request by jjti**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,10 +3,7 @@
   <span>Consul Telemetry Collector</span>
 </h1>
 
-Consul Telemetry Collector is a lightweight OpenTelemetry collector used to
-collect metrics, logs and traces from a Consul cluster and collect metrics from envoy
-service proxies and export them to HCP or another OTLP compliant endpoint.
-The metric sink is encrypted and authorized by the Consul service mesh.
+Consul Telemetry Collector is a lightweight [OpenTelemetry Collector](https://opentelemetry.io/docs/collector/) used to collect metrics from Envoy proxies and export them to HCP and/or another OTLP compliant metrics endpoint. The [Envoy stats sink](https://www.envoyproxy.io/docs/envoy/latest/api-v3/config/metrics/v3/metrics_service.proto#extension-envoy-stat-sinks-metrics-service) is encrypted and authorized by the Consul service mesh.
```
Comment (jjti) [inline README.md:6]: main change here is to remove reference to collecting server metrics, logs, traces. Also, add a link to the envoy stats sink docs

File: README.md [hunk 2]
```diff
@@ -16,52 +13,60 @@ Configuration will be loaded in the following order of precedence:
...
-HCP_CLIENT_ID=<client_id> HCP_CLIENT_SECRET=<client_secret> HCP_RESOURCE_ID=<resource_id> consul-telemetry-collector agent
+  -hcp-resource-id=<string>
+     HCP Resource ID Environment variable HCP_RESOURCE_ID
+
+  -http-collector-endpoint=<string>
+     OTLP HTTP endpoint to forward telemetry to Environment variable
+     CO_OTEL_HTTP_ENDPOINT
 ```
 
-## Development
```
Comment (jjti) [inline README.md:46]: current development is under installation

File: README.md [hunk 3]
```diff
@@ -16,52 +13,60 @@ Configuration will be loaded in the following order of precedence:
...
 # Installation
 
-To install and use the Consul Telemetry Collector you will need a Consul version of 1.15.3 or greater and to authorize communication with the collector on the Service Mesh.
+Documentation on deploying the Consul Telemetry Collector to a Kubernetes runtime are provided in the HCP documentation:
 
-## Deployment instructions
+- [Deploy the Consul telemetry collector](https://developer.hashicorp.com/hcp/docs/consul/monitor/management-plane/observability/telemetry-collector)
 
-### Docs
+## Configuration
```
Comment (jjti) [inline README.md:14]: making a dedicated config section that includes `-help` output to show everything

File: README.md [hunk 4]
```diff
@@ -3,65 +3,70 @@
   <span>Consul Telemetry Collector</span>
 </h1>
 
-Consul Telemetry Collector is a lightweight OpenTelemetry collector used to
-collect metrics, logs and traces from a Consul cluster and collect metrics from envoy
-service proxies and export them to HCP or another OTLP compliant endpoint.
-The metric sink is encrypted and authorized by the Consul service mesh.
+Consul Telemetry Collector is a lightweight [OpenTelemetry Collector](https://opentelemetry.io/docs/collector/) used to collect metrics from Envoy proxies and export them to HCP and other OTLP compliant metrics endpoints. The [Envoy stats sink](https://www.envoyproxy.io/docs/envoy/latest/api-v3/config/metrics/v3/metrics_service.proto#extension-envoy-stat-sinks-metrics-service) is encrypted and authorized by the Consul service mesh.
```
Comment (jjti) [inline README.md:6]: this is the main and most important change: we don't collect anything from servers right now. Especially not logs/traces

File: README.md [hunk 5]
```diff
@@ -3,65 +3,70 @@
...
+  -hcp-client-secret=<string>
+     HCP Service Principal Client Secret Environment variable
+     HCP_CLIENT_SECRET
 
-```bash
-HCP_CLIENT_ID=<client_id> HCP_CLIENT_SECRET=<client_secret> HCP_RESOURCE_ID=<resource_id> consul-telemetry-collector agent
+  -hcp-resource-id=<string>
+     HCP Resource ID Environment variable HCP_RESOURCE_ID
+
+  -http-collector-endpoint=<string>
```
Comment (Achooo) [inline README.md:46]: should we remove this? And just keep it forward to HCP specific?

**Pull request merged**